apps/web/src/components/Activity/PoolActivityItem.tsx

```diff
@@ -38,6 +43,57 @@ function updatedAfterPublish(
   return isAfter(parseDateTimeUtc(createdAt), parseDateTimeUtc(publishedAt));
 }
 
+const keyMap = new Map<string, MessageDescriptor>([
+  ["name", commonMessages.name],
+  ["user_id", adminMessages.user],
+  ["operational_requirements", navigationMessages.workPreferences],
+  ["key_tasks", jobPosterTemplateMessages.keyTasks],
+  ["closing_date", processMessages.closingDate],
+  ["your_impact", processMessages.yourImpact],
+  ["advertisement_location", processMessages.location],
+  ["security_clearance", processMessages.securityRequirement],
+  ["advertisement_language", processMessages.languageRequirement],
+  [
+    "is_remote",
+    defineMessage({
+      defaultMessage: "Remote",
+      id: "4vScqR",
+      description: "Label for when a position is remote",
+    }),
+  ],
+  ["process_number", processMessages.processNumber],
+  ["publishing_group", processMessages.publishingGroup],
+  [
+    "published_at",
+    defineMessage({
+      defaultMessage: "Published the advertisement",
+      id: "VltUUH",
+      description: "Message for publishing a process in the activity log",
+    }),
```

Comment:
Per this part of the issue, I am not sure we actually want to use the message "Published the advertisement". It was an example of what to do

<img width="495" height="123" alt="Image" src="https://github.com/user-attachments/assets/ed85ab93-eebf-4409-a492-af6643124036" />

The existing copy applicable if I was to guess would be 
```
    defaultMessage: "Published",
    id: "BcEpFb",
    description: "Status is published",
```

Reply:
Okay